dashboard/app/models/concept.rb

```diff
@@ -56,11 +53,17 @@ def self.setup
   def self.setup_with_concepts(concepts_by_index)
     videos_by_concept = Video.where(key: concepts_by_index).index_by(&:key)
     concepts = concepts_by_index.map.with_index(1) do |concept, id|
-      {id: id, name: concept, video_id: videos_by_concept[concept]&.id}
+      {id: id, name: concept, video_key: videos_by_concept[concept]&.key}
     end
     transaction do
       reset_db
       Concept.import! concepts
     end
   end
+
+  def related_video
+    Rails.cache.fetch('concepts/videos/{video_key}/{I18n.locale.to_s}') do
```

Comment:
is this supposed to be

```suggestion
    Rails.cache.fetch("concepts/videos/#{video_key}/#{I18n.locale.to_s}") do
```